modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClosureAngularClientCodegen.java

```diff
@@ -64,15 +70,11 @@ public JavascriptClosureAngularClientCodegen() {
 
         typeMapping.put("binary", "string");
 
-        outputFolder = "generated-code/javascript-closure-angular";
-        modelTemplateFiles.put("model.mustache", ".js");
-        apiTemplateFiles.put("api.mustache", ".js");
-        embeddedTemplateDir = templateDir = "Javascript-Closure-Angular";
-        apiPackage = "API.Client";
-        modelPackage = "API.Client";
-
         cliOptions.add(new CliOption(CodegenConstants.HIDE_GENERATION_TIMESTAMP, "hides the timestamp when files were generated")
                 .defaultValue(Boolean.TRUE.toString()));
+        cliOptions.add(new CliOption(USE_ES6,
+                "use plain ES6 templates instead - drops Google Closure Compiler annotations")
```

Comment:
That's fair, buuuuut the generated code is meant to part of an ES6 compilation process and Closure compiler doesn't support ES6 as output. [https://github.com/google/closure-compiler/wiki/ECMAScript6](url) 

My process goes like this: swagger generates the client code for ES6-Angular, I then move  the generated code into the project that requires it, and run my build process on that (webpack, gulp, whatever). The main assumptions of this feature are that the user already has a build process in place, and they are using Angular Resources. 

Understandably maybe this feature doesn't fall under Closure-Angular, however, there are no plain Javascript-Angular modules to work this into. Do you think this would be better expanded into it's own module?